src/CodeGen_Hexagon.h

```diff
@@ -24,8 +24,8 @@ class CodeGen_Hexagon : public CodeGen_Posix {
 
     void init_module();
 
-    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer);
-    void pop_buffer(const std::string &name, int dimensions);
+    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer, bool global = false);
```

Comment:
I agree, but it sounds like if we start using override, we need to use it in entire classes and that seems to be tricky. I'd like tor resolve when and where to use override separately.